share/doc/DEVELOPERS_GUIDE.md

```diff
@@ -404,3 +406,16 @@ in place of **pcc64le**. You could also replace the **epel-7-ppc64le** os-archit
 if you simply want to build natively for instance if you are running fedora on
 an **x86_64** machine the `frocearch pcc64le` in the above case could just be
 dropped. 
+
+
+## Units in VOTCA
+
+VOTCA tried as much as possible to standarize units across both CSG and XTP. Internally we use:
+
+- `nm` for molecular dynamics and coarse grained positions. `Bohr` is used for quantum mechanical and electrostatic coordinates in XTP
```

Comment:
Well, the internal units are `nm`, but we read whatever the spec of the format says, e.g. `pdb` is in Angstrom and `xyz` is in `nm`.

Reply:
I have updated the lammps parser so that they assume the lammps files are in units of 'units real', so there is no need to do pre-processing of the files before passing them to votca. You can see it here: https://github.com/votca/csg/pull/355

That pull request also makes assumptions about the other units in the file readers, e.g. dlpoly and others.

Reply:
I think we should say that readers take in the unit of their respective specification.